CHANGELOG.md

```diff
@@ -29,6 +29,14 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+
+## 0.??.?
```

Comment:
Not sure about the version number, do we need it at all?

Reply:
Afaik, yes we do.

Reply:
What should I put? `0.37.2a1`?

Reply:
Javid said it's not needed 
https://github.com/up42/up42-py/pull/585#discussion_r1565185019